### DISCUSSION TITLE
Welcome to cbet-resource-hub-frontend Discussions!

### DISCUSSION THREAD

**Discussion by anamboijohn** [acesknust/cbet-resource-hub-frontend#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## UI we'll be building

![WhatsApp Image 2024-04-30 at 13 00 50_6094a6c5](https://github.com/acesknust/cbet-resource-hub-frontend/assets/65270082/505476bc-4b33-4df5-977b-a976a5db3c91)

**Comment by PapaYawAsiamah** [comment]
looks good

**Reply by anamboijohn** [reply]
that's fine

**Reply by anamboijohn** [reply]
so let's deal with them one after the other

**Reply by anamboijohn** [reply]
you can create an Issue to work on the Landing page, the first one

**Reply by PapaYawAsiamah** [reply]
sure thing

**Reply by IsaacHevi** [reply]
Hello 
I'm Isaac
Is it one page for a person?

**Reply by PapaYawAsiamah** [reply]
Yea i think so. so you create an issue, assign yourself and create a branch for the feature you are working on

**Comment by PapaYawAsiamah** [comment]
My name is Papa Yaw Owusu Asiamah. I will be helping to build this UI

**Reply by anamboijohn** [reply]
Great to have you Papa Yaw

**Reply by IsaacHevi** [reply]
That's great
Nice to meet you
